src/libraries/Microsoft.Extensions.Configuration.Xml/tests/ConfigurationProviderXmlTest.cs

```diff
@@ -75,7 +82,7 @@ private void SectionToXml(StringBuilder xmlBuilder, string sectionName, TestSect
                 {
                     for (var i = 0; i < tuple.Value.AsArray.Length; i++)
                     {
-                        xmlBuilder.AppendLine($"<{tuple.Key} Name=\"{i}\">{tuple.Value.AsArray[i]}</{tuple.Key}>");
```

Comment:
Why is this changing? Is this a breaking change?

Reply:
In the past, the 'Name' attribute was required to support arrays. Now, this is no longer required. I felt it was more appropriate to use the simplest solution for the test suite instead of depending on a workaround that is no longer required.

To be clear, the 'Name' attribute is still completely supported, so I don't consider it a breaking change.

Reply:
Sounds good. Do we want to add a test that includes the `Name` attribute just to make sure we don't regress that support on the future?

Reply:
I believe it should already be covered by the existing tests, but I'll add a test that explicitly tests that the Name attribute can be used to simulate arrays.

Reply:
Done